### PR TITLE
New data

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
MT 2010 updates:
image

File: scripts/process/histWuClean.R [hunk 1]
```diff
@@ -6,8 +6,23 @@ process.histWuClean <- function(viz){
...
+  wuData <- bind_rows(wuData, waterMT)
+  
+  mt_2010 <- filter(wuData, 
+                    state_cd == "30",
+                    year == 2010) %>%
+    filter(!is.na(value))
+  
+  mt_2010_tots <- mt_2010[1,]
+  mt_2010_tots$category[1] <- "Total"
+  mt_2010_tots$value[1] <- sum(mt_2010$value)
```
Comment (lindsayplatt) [inline scripts/process/histWuClean.R:20]: could these MT specific things happen in the MT clean script? that way we can easily change back to pulling from NWIS in the future if we need to?

**Comment by jordansread** [comment]
We should gut-check the totals w/ WU (or the compilation reports) to make sure it makes sense that 2010 is such a dip in WU compared to 2005 and 2015. 
👍

**Comment by lindsayplatt** [comment]
Can we merge and get this out on dev, then send to verify? We won't be going to prod until the 2015 data is actually released anyways

File: scripts/process/histWuClean.R (hunk 1 above)
Comment (ldecicco-USGS) [reply scripts/process/histWuClean.R:20]: There's some MT data in the wuData that we need, so no...it's needs to be done here.

**Comment by ldecicco-USGS** [comment]
It's up on dev. One somewhat good sign is the dip in 2010  is also a dip on our original vizzy

**Pull request merged**